draft-ietf-quic-tls.md

```diff
@@ -532,9 +532,10 @@ older than 1.3 is negotiated.
 
 ## ClientHello Size
 
-QUIC requires that the initial handshake packet from a client fit within a
-single packet of at least 1280 octets.  With framing and packet overheads this
-value could be reduced.
+QUIC requires that the initial handshake packet from a client fit within the
+payload of a single packet.  A minimum-sized initial packet in IPv6 has space
+for a single TLS record of 1197 octets once the QUIC header and minimal framing
+is added.
```

Comment:
This is phrased awkwardly, and also seems to semi-conflict with {{#packetization}}.

I would consolidate these, and you also need to fix the requirement in {{#packetization}} that things be exactly 1280 because the client has no control over that.

I think rather what we need to say is:

1. The client needs to ensure that the initial packet has at least X bytes of UDP payload
2. When you add the QUIC header, that means CH can be up to Y bytes.
3. The rest of the space MUST be used by PADDING frames.

Based on the text, I believe that X == 1232 (I wouldn't bother to adjust between v4 and v6).
How do you get 1197? If we have 1232 bytes for UDP payload and then a 17-byte minimal QUIC header and a 3 byte STREAM frame header....
   
   
And I would put all of this in this space and leave the general guidance in {{#packetization}}.